app/src/main/res/values/strings.xml

```diff
@@ -210,11 +212,35 @@
 
     <!-- Spinner Download Size array-->
     <string-array name="download_size_array">
+        <item>Square</item>
+        <item>Thumbnail</item>
+        <item>Small</item>
+        <item>Medium</item>
+        <item>Large</item>
+        <item>Extra Large</item>
+    </string-array>
+
+    <!-- Spinner Download Size array (should stay untranslated) -->
+    <string-array name="download_size_array_values">
```

Comment:
this should have attribute translatable=false, shouldn't it?

Reply:
Done in https://github.com/Piwigo/Piwigo-Android/pull/201/commits/b632c4ff9a3f9313a7c4e96a60897da733d3439f